dev/com.ibm.ws.anno_fat/build.gradle

```
@@ -23,4 +26,22 @@ dependencies {
     'org.apache.httpcomponents.client5:httpclient5:5.0.2',
     'org.apache.httpcomponents.core5:httpcore5:5.0.2',
     project(':io.openliberty.org.apache.commons.codec')
+  springLibs 'org.springframework:spring-beans:6.1.14', 
+	'org.springframework:spring-context:6.1.14',
+	'org.springframework:spring-core:6.1.14',
+	'org.springframework:spring-expression:6.1.14',
+	'org.springframework:spring-web:6.1.14',
+	'org.springframework:spring-webmvc:6.1.14',
+	 project(':io.openliberty.org.apache.commons.logging')
```

```suggestion
	'org.springframework:spring-webmvc:6.1.14'
```
Already part of jar in fattest.simplicity that are on the classpath